pkg/controller/directvolumemigration/namespace.go

```diff
@@ -22,13 +23,14 @@ func (t *Task) ensureDestinationNamespaces() error {
 	}
 
 	// Get list namespaces to iterate over
-	nsMap := t.getPVCNamespaceMap()
-	for ns, _ := range nsMap {
+	nsMap := t.getBothPVCNamespaceMap()
+	for bothNs, _ := range nsMap {
+		nsNames := strings.Split(bothNs, ":")
 		// Get namespace definition from source cluster
 		// This is done to get the needed security context bits
 
 		srcNS := corev1.Namespace{}
-		key := types.NamespacedName{Name: ns}
+		key := types.NamespacedName{Name: nsNames[0]}
```

Comment:
Sure. The way we call this basically guarantees we'll have "src:target", even if they're the same in this context, but it can't hurt to add the additional check. If len(nsNames) is 1 we can assume dest name == nsnames[0]. I'll update it.